docs/samples/Microsoft.ML.Samples/Dynamic/DataOperations/EntityFrameworkEnumerable.cs

```diff
@@ -0,0 +1,128 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Microsoft.EntityFrameworkCore;
+
+namespace Microsoft.ML.Samples.Dynamic
+{
+    public class HousingData
+    {
+        // Entity Framework requires the object to have a primary key
+        public int HousingDataId { get; set; }
+        public float MedianHomeValue { get; set; }
+        public float CrimesPerCapita { get; set; }
+        public float PercentResidental { get; set; }
+        public float PercentNonRetail { get; set; }
+        public float CharlesRiver { get; set; }
+        public float NitricOxides { get; set; }
+        public float RoomsPerDwelling { get; set; }
+        public float PercentPre40s { get; set; }
+        public float EmploymentDistance { get; set; }
+        public float HighwayDistance { get; set; }
+        public float TaxRate { get; set; }
+        public float TeacherRatio { get; set; }
+    }
+
+    public partial class MLNetExampleContext : DbContext
+    {
+        public DbSet<HousingData> Housing { get; set; }
+
+        public MLNetExampleContext(DbContextOptions<MLNetExampleContext> options)
+            : base(options)
+        {
+        }
+
+        protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
+        {
+            if (!optionsBuilder.IsConfigured)
```

Comment:
This is probably ok to remove. We add it when the context is reverse engineered so that customers that want to pass the DbContext configuration externally don't need to change the generated code.